src/mobile/components/layout/templates/scaffold.jade

```diff
@@ -39,5 +39,35 @@ html( class=htmlClass )
       //- Stripe
       script( type="text/javascript", src="https://js.stripe.com/v3/" )
 
+      //- Sailthru Horizon JS - user interest data, personalization
+      if sd.SAILTHRU_CUSTOMER_ID
```

Comment:
We weren't including this on mobile web before, perhaps because all of the pages that used it were responsive?